package.json

```diff
@@ -79,7 +79,10 @@
     "pre-commit": "lint-staged"
   },
   "lint-staged": {
-    "*": "trunk check --fix",
+    "*": [
+      "trunk fmt",
+      "trunk check --fix"
+    ],
```

Comment:
Isn't formatting already included in `trunk check --fix`?

Reply:
Oh! i didn't realise i sometimes use unstaged commits to avoid autoformating of rules.d.ts.

I think this is working properly.